modules/local/generate_input/main.nf

```diff
@@ -13,14 +12,20 @@ process GENERATE_INPUT {
         'quay.io/biocontainers/python:3.9' }"
 
     input:
-       path(merged_files) 
+    path(merged_files) 
 
     output:
-//    publishDir 'fastq_pass_merged', mode: 'move'
-      path '*amplesheet_merged.csv' , emit : sample_sheet_merged
+    // publishDir 'fastq_pass_merged', mode: 'move'
+    path '*amplesheet_merged.csv' , emit : sample_sheet_merged
```

Comment:
Should this be path `'*samplesheet_merged.csv'` ?

Reply:
Tbh I just left it as is. I think it should be named something more specific pertaining to the sequencing run name.